dbrequests/mysql/tests/conftest.py

```diff
@@ -0,0 +1,86 @@
+"""Configures the test environment. Some helper functions and fixtures."""
+import time
+
+import pytest
+from dbrequests.mysql import Database
+
+from docker import from_env
+from docker.errors import APIError
+
+
+def run_docker_container():
+    """Run mariadb-docker container and return proper url for access."""
+    creds = {
+        'user': 'root',
+        'password': 'root',
+        'host': '127.0.0.1',
+        'db': 'test',
+        'port': 3307
+    }
+    client = from_env()
+    try:
+        container = client.containers.run(
+            'mariadb:10.3',
+            name='test-mariadb-database',
+            ports={3306: creds['port']},
+            environment={
+                'MYSQL_ROOT_PASSWORD':
+                creds['password'],
+                'MYSQL_DATABASE': creds['db']
+            },
+            detach=True)
+        time.sleep(50)
+    except APIError:
+        container = client.containers.get('test-mariadb-database')
+
+    url = ("mysql+pymysql://{}:{}@{}:{}/{}".format(creds['user'],
+                                                   creds['password'],
+                                                   creds['host'],
+                                                   creds['port'], creds['db']))
+    return url, container
+
+
+def kill_remove_docker_container(container):
+    """Kill and remove mariadb docker container."""
+    container.kill()
+    container.remove()
+    return 0, 'Container {} removed.'.format(container.id[:12])
+
+
+def set_up_cats(db):
+    """Set up the cats table for testing."""
+    db.bulk_query("""
+        SET @@SQL_MODE = REPLACE(@@SQL_MODE, 'STRICT_TRANS_TABLES', '');
```

Comment:
is this something what should be in the other setup file as well?

Reply:
This is like `dbrequests.tests.conftest.set_up_table`. Will be gone if we solve the parameterized fixtures.